google-cloud-clients/google-cloud-iamcredentials/pom.xml

```diff
@@ -110,4 +68,4 @@
       </plugin>
     </plugins>
   </reporting>
-</project>
+</project>
```

Comment:
Please make sure to have a newline character at the bottom of file.

Reply:
If you see it is showing a difference in here.. meaning we have to add an empty line at the bottom of the file.